### DISCUSSION TITLE
Problems in how GIS and data can be used

### DISCUSSION THREAD

**Discussion by ghost** [albertkun/21S-ASIAAM-191A#20] [Past Thinking Caps]
Since being exposed to GIS I have been very connected to visual, especially cartographic, displays of qualitative and quantitative data. It feels like I learn something new every day whether it be finding a new data source or learning how to utilize a mapping analysis function. For me, it is easy to get caught up in the learning and absorption of information within these versatile tools. As my experience has grown I've been able to take more time to reflect on my employment of information and possible misuse due to convenience or just a complete misunderstanding of the data on my part. This is where I can see how misinformation, unequal power structures, etc can be so easily perpetuated on web platforms that utilize GIS or just data in general. I think, concerning data usage within GIS, beginners are the most likely to fall into unintentional uses or displays of information because they are so focused on what these tools can do rather than the harm they can also generate. Many people who are using such data do not have deep insight into its purpose/function and while using data second hand is a wonderful facet of the internet, it can also be damaging and potentially dangerous to certain groups of people. There are also a lot of ways in which data and GIS can be purposefully/intentionally employed to confuse, misdirect, or ignore important information within a dataset. We often see this within politics and I have especially noticed this concerning COVID-19 maps or statistics that are used to push a narrative that is often not in line with the true data or extent of the issue.

**Comment by rachan2023** [comment]
I agree that beginners are also susceptible to misuse of data and GIS because they aren't aware of the repercussions and how much harm they can do to certain communities. Data and GIS are important tools for providing information to the public, but many people are unconsciously and consciously using these tools to push a narrative, as you've said. One of the solutions that I see for alleviating this problem is teaching people how to use these tools ethically and encouraging people who have access to these tools to use them in ways that benefit underserved or underrepresented communities. It is also important to educate people on how to critically examine data visualizations, so that people are aware of the narratives that are being told as well as their consequences on society. Especially during the pandemic and the age of wide spread information, it is easy for people to use data to show one side of the story, which contributes to the dissemination of misinformation.

**Comment by latishasumardy** [comment]
I agree that it is very easy to misuse data, especially if you don't understand it. There are so many uses for a single set of data, and it can be very easy to get caught up in all the things that you can do with it, before even thinking about the possible implications of it. I also agree with the point that you made about GIS being used for nefarious purposes. I often see this in action on news stations, especially when you compare reports from different newsgroups. Even though the data that the groups use is the same, the way that they display it definitely pushes their own narrative, and if you have no background in this topic it can be very easy to immediately believe what they say without clearly understanding what the maps actually display.
